tests/vmi_configuration_test.go

```diff
@@ -204,41 +204,48 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 		)
 
 		DescribeTable("with memory configuration", func(vmiOptions []libvmi.Option, expectedGuestMemory int) {
```

Comment:
Thank you for addressing these tests. Both of them test simple vmi.yaml -> domxml conversion and should take place in a unit test. They don't merit the time and risk of flakiness that starting up a VM and logging into it bring with them.

Reply:
I agree, I'm a little concerned these changes could unnecessarily prolong the test run times and introduce additional flakiness. Can you convert them to unit tests instead please?